### PR TITLE
CW-423 Use SensitiveClipboard for sensitive Stings

### PR DESCRIPTION
**Pull request by konstantinullrich**
# Description

Use Android 13 SensitiveClipboard for sensitive Stings

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [x] Double check modified code and verify it with the feature/task requirements
- [x] Format code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -45,8 +45,8 @@ android {
 
     defaultConfig {
         applicationId appProperties['id']
-        minSdkVersion 21
-        targetSdkVersion 31
+        minSdkVersion 24
```
Comment (SamsungGalaxyPlayer) [inline android/app/build.gradle:48]: Should this be reverted to 21?

Comment (konstantinullrich) [reply android/app/build.gradle:48]: No, in order to set Extras to Clipboard descriptions it needs to be, at least, at API >= 24 (android N) :/

File: lib/src/screens/backup/backup_page.dart [hunk 2]
```diff
@@ -52,8 +52,7 @@ class BackupPage extends BasePage {
                       child: Observer(
                           builder: (_) => GestureDetector(
                                 onTap: () {
-                                  Clipboard.setData(
-                                      ClipboardData(text: backupViewModelBase.backupPassword));
+                                  SensitiveClipboard.copy(backupViewModelBase.backupPassword);
```
Comment (OmarHatem28) [inline lib/src/screens/backup/backup_page.dart:55]: check here if the platform is mobile then use this package if not, then use the old method

**Review by OmarHatem28 — CHANGES_REQUESTED** [review]
Instead of going through all the places where you used the new package and adding the required check (if platform is mobile) you can make a utility function that handles the copy and call it instead and you can write the condition only once there

you can check the `share_util.dart` it's made for a reason close to this

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**